### PR TITLE
MINOR: fix JavaDoc

### PR DESCRIPTION
**Pull request by mjsax**
Call for review @vvcephei

### CODE REVIEW

File: streams/src/main/java/org/apache/kafka/streams/errors/StreamsNotStartedException.java [hunk 1]
```diff
@@ -17,11 +17,12 @@
 package org.apache.kafka.streams.errors;
 
 import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.KafkaStreams.State;
 
 /**
- * Indicates that Kafka Streams is in state {@link KafkaStreams.State#CREATED CREATED} and thus state stores cannot be queries yet.
+ * Indicates that Kafka Streams is in state {@link State CREATED} and thus state stores cannot be queries yet.
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/errors/StreamsNotStartedException.java:23]: We can only link to the enum, but not its values.

File: streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java [hunk 2]
```diff
@@ -106,7 +105,7 @@ void register(final StateStore store,
 
     /**
      * Schedules a periodic operation for processors. A processor may call this method during
-     * {@link Processor#init(org.apache.kafka.streams.processor.ProcessorContext) initialization} or
+     * {@link Processor#init(ProcessorContext) initialization} or
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java:108]: Removing the package name to point to `org.apache.kafka.streams.processor.api.ProcessorContext` (note the `api` sub-package in the path)

File: streams/src/main/java/org/apache/kafka/streams/errors/StreamsNotStartedException.java (hunk 1 above)
Comment (vvcephei) [reply streams/src/main/java/org/apache/kafka/streams/errors/StreamsNotStartedException.java:23]: Ah!

File: streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java (hunk 2 above)
Comment (vvcephei) [reply streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java:108]: Good catch.

**Review by vvcephei — APPROVED** [review]
Thanks, @mjsax !

**Comment by mjsax** [comment]
Retest this please.

**Pull request merged**